src/compas/datastructures/mesh/offset.py

```diff
@@ -85,4 +130,4 @@ def mesh_thicken(mesh, thickness=1.0):
 if __name__ == '__main__':
 
     import doctest
-    doctest.testmod(globs=globals())
+    doctest.testmod()
```

Comment:
I think we can get rid of this whole `if __name__ ...` thing, right?

Reply:
am testing it now to see if sphinx doctesting still works (and fixing the doctests while at it). will send separate PR for that...